src/configs/lang.ts

```diff
@@ -409,9 +409,9 @@ export const WELCOME_BACK = translate('WELCOME_BACK', 'Welcome back');
 export const LINK = translate('LINK', 'Link');
 export const IRS_GREEN_THRESHOLD = translate('IRS_GREEN_THRESHOLD', 'Green');
 export const IRS_GREY_THRESHOLD = translate('IRS_GREY_THRESHOLD', 'Grey');
-export const IRS_RED_THRESHOLD = translate('RED_THRESHOLD', 'Red');
-export const IRS_YELLOW_THRESHOLD = translate('YELLOW_THRESHOLD', 'Yellow');
-export const IRS_LIGHT_GREEN_THRESHOLD = translate('LIGHT_GREEN_THRESHOLD', 'Light Green');
+export const IRS_RED_THRESHOLD = translate('IRS_RED_THRESHOLD', 'Red');
+export const IRS_YELLOW_THRESHOLD = translate('IRS_YELLOW_THRESHOLD', 'Yellow');
+export const IRS_LIGHT_GREEN_THRESHOLD = translate('IRS_LIGHT_GREEN_THRESHOLD', 'Light Green');
```

Comment:
@p-netm are these updated in `strings/en.json`?

Reply:
@cKellyDesign they did not require updating. https://github.com/onaio/reveal-frontend/blob/d1850b85283465441b0ce45569899745967de20e/src/configs/strings/en.json#L858.